### PR TITLE
implemented image, select and input ui components

### PR DESCRIPTION
**Pull request by Alimahmood7**

### CODE REVIEW

File: src/components/ui/image/image.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import {
+  Image,
+} from 'rebass';
+import Proptypes from 'prop-types';
+
+export const imageWrapper = styled(Image)``;
```
Comment (InamTaj) [inline src/components/ui/image/image.js]: Name of every react component must start with capital letter, eg: `ImageWrapper`

File: src/components/ui/input/input.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+import {
+  Input,
+} from '@rebass/forms';
+import Proptypes from 'prop-types';
+
+export const inputWrapper  = styled(Input)``;
```
Comment (InamTaj) [inline src/components/ui/input/input.js]: `InputWrapper`

File: src/components/ui/input/input.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+
+inputWrapper.propTypes = {
+  type: Proptypes.string.isRequired,
+  id: Proptypes.string.isRequired,
```
Comment (InamTaj) [inline src/components/ui/input/input.js]: Sort props alphabetically

File: src/components/ui/input/input.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  name: Proptypes.string.isRequired,
+};
+
```
Comment (InamTaj) [inline src/components/ui/input/input.js]: Why extra empty lines?

File: src/components/ui/select/index.js [hunk 5]
```diff
@@ -0,0 +1 @@
+export {selectWrapper as Select} from './select';
```
Comment (InamTaj) [inline src/components/ui/select/index.js]: Give spaces inside brackets

File: src/components/ui/select/select.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+import {
+  Select,
+} from '@rebass/forms';
+import Proptypes from 'prop-types';
+
+export const selectWrapper = styled(Select)``;
```
Comment (InamTaj) [inline src/components/ui/select/select.js]: `SelectWrapper`

File: src/components/ui/image/image.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import {
+  Image,
```
Comment (InamTaj) [inline src/components/ui/image/image.js]:
```
import { Image as rImage } from 'rebass';
```

and then just do the following:

`export const Image = styled(rImage)``;`

No need for additional Wrapper word with each component

Comment (InamTaj) [reply src/components/ui/image/image.js]: apply this rule on all files

File: src/components/ui/image/index.js [hunk 8]
```diff
@@ -0,0 +1 @@
+export { ImageWrapper as Image } from './image';
```
Comment (InamTaj) [inline src/components/ui/image/index.js]: After refactor, it will be `export { Image } from './image';`

File: src/components/ui/input/index.js [hunk 9]
```diff
@@ -0,0 +1 @@
+export { InputWrapper as Input } from './input';
```
Comment (InamTaj) [inline src/components/ui/input/index.js]: `export { Input } from './input';`

File: src/components/ui/input/input.js [hunk 10]
```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+import {
+  Input,
```
Comment (InamTaj) [inline src/components/ui/input/input.js]: `import { Input as rInput } from '@rebass/forms';`

File: src/components/ui/input/input.js (hunk 3 above)
Comment (InamTaj) [reply src/components/ui/input/input.js]: still unaddressed

File: src/components/ui/select/select.js [hunk 11]
```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+import {
+  Select,
+} from '@rebass/forms';
+import Proptypes from 'prop-types';
+
+export const SelectWrapper = styled(Select)``;
```
Comment (InamTaj) [inline src/components/ui/select/select.js]: `export const Select = styled(rSelect);`

File: src/components/ui/input/input.js (hunk 3 above)
Comment (Alimahmood7) [reply src/components/ui/input/input.js]: I have sort this with the extension sort lines in vs code and is not sort in ascending way alphabetically like i comes before then n and then t  ?

**Pull request merged**